trader/benchmark.py

```diff
@@ -0,0 +1,116 @@
+import typer
+from prediction_market_agent_tooling.benchmark.agents import (
+    AbstractBenchmarkedAgent,
+    RandomAgent,
+)
+from prediction_market_agent_tooling.benchmark.benchmark import Benchmarker
+from prediction_market_agent_tooling.benchmark.utils import (
+    OutcomePrediction,
+    Prediction,
+)
+from prediction_market_agent_tooling.gtypes import Probability
+from prediction_market_agent_tooling.loggers import logger
+from prediction_market_agent_tooling.markets.markets import (
+    FilterBy,
+    MarketType,
+    SortBy,
+    get_binary_markets,
+)
+from prediction_prophet.benchmark.agents import _make_prediction
+from dotenv import load_dotenv
+from trader.prediction import DEFAULT_MODEL, predict
+
+
+def main(
+    n: int = 10,
+    output: str = "./benchmark_report.md",
+) -> None:
+    # Load environment variables
+    load_dotenv()
+
+    # Get n markets from https://manifold.markets, where real humans are trading.
+    markets = get_binary_markets(
+        n, MarketType.MANIFOLD, filter_by=FilterBy.OPEN, sort_by=SortBy.CLOSING_SOONEST
+    )
+    markets_deduplicated = list(({m.question: m for m in markets}.values()))
+    if len(markets) != len(markets_deduplicated):
+        logger.info(
+            f"Warning: Deduplicated markets from {len(markets)} to {len(markets_deduplicated)}."
+        )
+
+    logger.info(f"Found {len(markets_deduplicated)} markets.")
+
+    benchmarker = Benchmarker(
+        markets=markets_deduplicated,
+        agents=[
+            RandomAgent(agent_name="random"),
+            QuestionOnlyAgent(),
+            PredictionAgent(),
```

Comment:
You could make it clear in the naming that this is the agent belonging to the participant 
```suggestion
            PredictionAgent(agent_name="**your agent**"),
```

Reply:
renamed, just on a different place